macro-plantuml-macro/src/main/java/org/xwiki/contrib/plantuml/PlantUMLMacroParameters.java

```diff
@@ -42,6 +44,16 @@ public void setServer(String serverURL)
         this.serverURL = serverURL;
     }
 
+    /**
+     * @param format see {@link #getFormat()}. Valid values: svg, png, txt
+     *
+     */
+    @PropertyDescription("the PlantUML output format")
```

Comment:
Missing examples here. This is is important for users since this is what they'll see in the WYSIWYG macro UI